src/core/include/units/math.h

```diff
@@ -147,4 +147,43 @@ template<Quantity Q>
   return Q(std::numeric_limits<typename Q::rep>::epsilon());
 }
 
+/**
+ * @brief Returns the greatest quantity that is representable in Unit To that is less or equal to q.
+ *
+ * The returned value is rounded down to a quantity with unit type To
+ *
+ * @tparam q Quantity type being the base of the operation
```

Comment:
```suggestion
 * @tparam q Quantity type being the base of the operation
```
Isn't `tparam` for template parameters? `q` is a value, not a type.